test/runtests.jl

```diff
@@ -1,13 +1,136 @@
-using OpenAIGym
-using Base.Test
+using OpenAIGym, Compat.Test
```

Comment:
I think `using Test` is fine now

Reply:
I'll leave you to do this and any other changes you like